app/src/lib/editors/launch.ts

```diff
@@ -32,10 +35,45 @@ export async function launchExternalEditor(
   if (editor.usesShell) {
     spawn(`"${editorPath}"`, [`"${fullPath}"`], { ...opts, shell: true })
   } else if (__DARWIN__) {
```

Comment:
NIT: Because of this guard diff double clicks will not go to the specified line.